Apps/Encounter/src/Delegates/RestMSPVisitDelegate.cs

```diff
@@ -108,7 +109,7 @@ public async Task<RequestResult<MSPVisitHistoryResponse>> GetMSPVisitHistoryAsyn
                 try
                 {
                     string json = JsonSerializer.Serialize(request, options);
-                    using HttpContent content = new StringContent(json);
+                    using HttpContent content = new StringContent(json, null, MediaTypeNames.Application.Json);
```

Comment:
content type was being set as plain/text, updated to the correct "app/json", ODR works with both